doc/man1/flux-job.rst

```diff
@@ -379,5 +379,5 @@ RESOURCES
 
 Flux: http://flux-framework.org
 
-RFC 34: Flux Task Map: https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_34.html
+:doc:`rfc:spec_34`
```

Comment:
I don't understand this sentence in the commit message:

> The man reference is now just the title but man doesn't support hyperlinks and the title can be googled

Oh, does this mean the reference to RFC 34 in the manpage when using `man(1)` is only a title?

Reply:
oops, my comment says it is outdated, so feel free to ignore if I commented on an older version of this PR

Reply:
No that is fine, it just wasn't clear (to me) from the commit message (maybe just change to "The reference in the man page is now just the RFC title ..." though not necessary if you'd rather not IMO)